src/main/scala/gitbucket/core/controller/api/ApiRepositoryContentsControllerBase.scala

```diff
@@ -13,10 +13,13 @@ import scala.util.Using
 trait ApiRepositoryContentsControllerBase extends ControllerBase {
   self: ReferrerAuthenticator with WritableUsersAuthenticator with RepositoryCommitFileService =>
 
-  /*
+  /**
    * i. Get the README
    * https://developer.github.com/v3/repos/contents/#get-the-readme
```

Comment:
```suggestion
   * i. Get a repository README
   * https://docs.github.com/en/rest/reference/repos#get-a-repository-readme
```

Reply:
fixed in https://github.com/gitbucket/gitbucket/pull/2535/commits/10d97a43f9760496ed1a73c580afbe2d4ee10c2b